resources/model/OpenStudio.idd

```diff
@@ -35695,19 +35694,19 @@ OS:UtilityBill,
        \type choice
        \required-field
        \key Electricity
-       \key Gas
+       \key NaturalGas
        \key Gasoline
        \key Diesel
        \key Coal
-       \key FuelOil_1
-       \key FuelOil_2
+       \key FuelOilNo1
+       \key FuelOilNo2
        \key Propane
        \key Water
-       \key Steam
+       \key DistrictHeatingSteam
        \key DistrictCooling
-       \key DistrictHeating
-       \key OtherFuel_1
-       \key OtherFuel_2
+       \key DistrictHeatingWater
+       \key OtherFuel1
+       \key OtherFuel2
```

Comment:
Hmm, how were these able to slip through the cracks?

Reply:
UtilityBill actually uses `FuelType` and FuelType::valueDescription to write the E+ IDF, so it was working just fine already

```c++
    explicit UtilityBill(const FuelType& fuelType, const Model& model);
```

I'm just aiming for consistency here.